main.go

```diff
@@ -122,35 +122,34 @@ func EchoHandler(writer http.ResponseWriter, request *http.Request) {
 
 	logger.Info("Echoing back request made to " + request.URL.Path + " to client (" + request.RemoteAddr + ")")
 
-	writer.Header().Set("Access-Control-Allow-Origin", "*")
-
 	// allow pre-flight headers
 	writer.Header().Set("Access-Control-Allow-Headers", "Content-Range, Content-Disposition, Content-Type, ETag")
 
 	request.Write(writer)
 }
 
-func SlightEchoHandler(writer http.ResponseWriter, request *http.Request) {
-	logger.Info("Slightly echoing back request made to " + request.URL.Path + " to client (" + request.RemoteAddr + ")")
+type Payload struct {
+	Message string `json:"message"`
+}
 
-	writer.Header().Set("Access-Control-Allow-Origin", "*")
-	writer.Header().Set("Access-Control-Allow-Headers", "Content-Range, Content-Disposition, Content-Type, ETag")
+func SlightEchoHandler(writer http.ResponseWriter, request *http.Request) {
+	log.Info("Slightly echoing back request made to " + request.URL.Path + " to client (" + request.RemoteAddr + ")")
```

Comment:
Just FYI:  you can use `logger.Infof("slightly echoing back request made to %s to client", request.URL.Path)`.